frontend/src/components/document/Comments/SidebarComment.jsx

```diff
@@ -68,18 +77,32 @@ function SidebarComment ({id, comment}) {
     )
 
     return (
-        <div onClick={handleOnClick} className={`flex flex-col justify-left p-1 bg-white p-5 rounded-2xl w-64 ${isFocus ? 'shadow-xl -ml-2.5 transform -translate-x-2.5' : ''}`}>
-            <div className="flex flex-row justify-between">
-                <div className="flex flex-row items-center space-x-2.5 mb-3">
-                    <img src={ProfileIcon} alt="Profile Pic" className="h-10"/>
-                    <div className="flex flex-col">
-                        <p className="font-semibold">{comment.author}</p>
-                        <p className="font-light">{format(new Date(), "MMM dd h:mmaa")}</p>
+        <div onClick={handleOnClick} className={`flex flex-col bg-none justify-left rounded-2xl ${isFocus ? 'shadow-xl -ml-2.5 transform -translate-x-2.5' : ''}`}>
+            <div className={`flex flex-col bg-white p-5 w-64 ${isFocus && isViewReply ? 'rounded-tl-2xl rounded-tr-2xl' : 'rounded-2xl'}`}>
+                <div className="flex flex-row justify-between">
+                    <div className="flex flex-row items-center space-x-2.5 mb-3">
+                        <img src={ProfileIcon} alt="Profile Pic" className="h-10"/>
+                        <div className="flex flex-col">
+                            <p className="font-semibold">{comment.author}</p>
+                            <p className="font-light">{format(new Date(), "MMM dd h:mmaa")}</p>
+                        </div>
                     </div>
+                {isFocus && menu }
+                
+                </div>
+                {comment.text}
+                <div onClick={toggleViewReply}>
+                    <p className="text-xs mt-2 font-light text-dark-grey underline decoration-light-gray-background">
+                        {isFocus ? (isViewReply ? "Hide Reply" : "View Reply") : ''}
+                    </p>
                 </div>
-            {isFocus && menu }
             </div>
-            {comment.text}
+            {isViewReply && isFocus && 
+                <div className="flex flex-col">
+                    <CommentChild />
```

Comment:
Does this mean only one child per comment rn? and it's prepopulated?

Reply:
yeahh i was going to populate them in after doing functionality